### PR TITLE
Logistics marshall delivery state

### PR DESCRIPTION
**Pull request by pmasilva20**

### CODE REVIEW

**Review by p-laranjinha — APPROVED** [review]

File: mainApp/src/main/java/logisticsmarshall/tqs/ua/services/DeliveryService.java [hunk 1]
```diff
@@ -102,8 +102,12 @@ private Driver validateDeliveryChange(User user, Delivery delivery, boolean shou
         Driver driver = user.getDriver();
         if (driver == null
                 || driver.getPhoneNo().isEmpty()
-                || Boolean.FALSE.equals(driver.getStatus()))
+                || Boolean.TRUE.equals(driver.getStatus()))
```
Comment (buckaroo69) [inline mainApp/src/main/java/logisticsmarshall/tqs/ua/services/DeliveryService.java:105]: this is a lowercase boolean now, please simplify

**Review by buckaroo69 — CHANGES_REQUESTED** [review]
no toStrings or hashcodes needed, also please clean system.out usage

File: mainApp/src/main/java/logisticsmarshall/tqs/ua/services/DeliveryService.java (hunk 1 above)
Comment (pmasilva20) [reply mainApp/src/main/java/logisticsmarshall/tqs/ua/services/DeliveryService.java]: done

**Comment by pmasilva20** [comment]
> please clean system.out usage

Also done I think

**Review by buckaroo69 — APPROVED** [review]
very cool

**Pull request merged**